src/Compilers/CSharp/Test/Symbol/Symbols/Retargeting/RetargetingTests.cs

```diff
@@ -845,6 +921,41 @@ public void CheckNamedTypes(NamedTypeSymbol a, NamedTypeSymbol b)
             CheckSymbols(a.TypeParameters, b.TypeParameters, true);
         }
 
+        public void CheckFunctionPointerTypes(FunctionPointerTypeSymbol a, FunctionPointerTypeSymbol b)
+        {
+            FunctionPointerUtilities.CommonVerifyFunctionPointer(a);
+            FunctionPointerUtilities.CommonVerifyFunctionPointer(b);
+            CheckSymbols(a.Signature.ReturnTypeWithAnnotations, b.Signature.ReturnTypeWithAnnotations, recurse: true);
+            checkModifiers(a.Signature.RefCustomModifiers, b.Signature.RefCustomModifiers);
+
+            Assert.Equal(a.Signature.ParameterCount, b.Signature.ParameterCount);
+
+            for (int i = 0; i < a.Signature.ParameterCount; i++)
+            {
+                var aParam = a.Signature.Parameters[i];
+                var bParam = b.Signature.Parameters[i];
+                CheckSymbols(aParam.TypeWithAnnotations, bParam.TypeWithAnnotations, recurse: true);
+                checkModifiers(aParam.RefCustomModifiers, bParam.RefCustomModifiers);
+            }
+
+            void checkModifiers(ImmutableArray<CustomModifier> aModifiers, ImmutableArray<CustomModifier> bModifiers)
```

Comment:
Believe this can be `static`

Reply:
No, I call `CheckSymbols`. That's an instance method.